content/sensu-go/5.14/getting-started/learn-sensu.md

```diff
@@ -150,95 +145,95 @@ You should see a confirmation message from sensuctl.
+Now that you're generating Slack alerts, let's reduce the potential for alert fatigue by adding a filter that sends only warning, critical, and resolution alerts to Slack.
 
-To accomplish this, we'll interactively add the built-in is_incident filter to the keepalive handler so we'll only receive alerts when the sandbox entity fails to send a keepalive event.
+To accomplish this, you'll interactively add the built-in is_incident filter to the keepalive handler, which will make sure you only receive alerts when the sandbox entity fails to send a keepalive event.
 
 {{< highlight shell >}}
 sensuctl handler update keepalive
 {{< /highlight >}}
 
-When prompted for the filters selection, enter `is_incident` to apply the incidents filter.
+The first prompt will be for environment variables. Just press `return` to continue. The second prompt is for the filters selection--enter `is_incident` to apply the is_incident filter.
 
 {{< highlight shell >}}
-? Filters: [? for help] is_incident
+? Filters: is_incident
 {{< /highlight >}}
 
-We can confirm that the keepalive handler now includes the incidents filter using sensuctl:
+For the mutator, timeout, type, runtime assets, and command prompts, just press `return` again.
+
+Use sensuctl to confirm that the keepalive handler now includes the is_incident filter:
 
 {{< highlight shell >}}
-sensuctl handler info keepalive
+[sensu_go_sandbox]$ sensuctl handler info keepalive
 === keepalive
 Name:                  keepalive
 Type:                  pipe
 Timeout:               0
 Filters:               is_incident
+Mutator:               
+Execute:               RUN:  sensu-slack-handler -c "${KEEPALIVE_SLACK_CHANNEL}" -w "${KEEPALIVE_SLACK_WEBHOOK}"
+Environment Variables: KEEPALIVE_SLACK_WEBHOOK=https://hooks.slack.com/services/ABC, KEEPALIVE_SLACK_CHANNEL=#monitoring
+Runtime Assets:        sensu-slack-handler
 {{< /highlight >}}
 
-With the filter in place we should no longer be receiving messages in the Slack channel every time the sandbox entity sends a keepalive event.
+With the filter in place, you should no longer receive messages in your Slack channel every time the sandbox entity sends a keepalive event.
 
-Let's stop the agent and confirm that we receive the expected warning message.
+Let's stop the agent and confirm that your receive the expected warning message.
 
 {{< highlight shell >}}
 sudo systemctl stop sensu-agent
 {{< /highlight >}}
 
-You should see the warning message in Slack after a couple of minutes, informing you that the sandbox entity is no longer sending keepalive events.
+After a couple minutes, you should see a warning message in your Slack channel informing you that the sandbox entity is no longer sending keepalive events.
 
-Before we go, start the agent to resolve the warning.
+Start the agent to resolve the warning.
```

Comment:
So there might be another enhancement here. I'll have to run through the sandbox to confirm, but it is possible that applying only the `is_incident` filter without the `is_resolution` one will only allow warns, critical or unknown alerts through and not the resolution. Up on line 187 it is mentioned that resolutions will come through, so I think I'll defer that away from this PR and confirm it once this is merged and I've gone through the entire thing.

Reply:
OK, thank you!

Reply:
I confirmed resolutions come through. Resolving.